setup.py

```diff
@@ -36,7 +36,7 @@ def str_from_file(name):
 long_description = str_from_file("README.rst")
 
 # tuples of (major, minor) of supported Python versions ordered from lowest to highest
-supported_python_versions = [(3, 8), (3, 9)]
+supported_python_versions = [(3, 8), (3, 9), (3, 10)]
```

Comment:
Just as a thought (no need at all to address this here): As `.ci/variables.json` became our source of truth regarding Python versions I wonder whether we should pick up the supported versions from there (may require us to change the file format a bit).

Reply:
Good idea, will open a follow-up pull request once this one is merged. :+1: